library/main.js

```diff
@@ -157,11 +180,11 @@ com.geertwille.main = {
         }
     },
 
-    copyLayerWithFactor: function(originalSlice, factor) {
+    copyLayerWithFactor: function(originalSlice, factor, densityScale) {
         var copy     = [originalSlice duplicate],
             frame    = [copy frame],
-            rect     = [[copy absoluteRect] rect];
```

Comment:
I removed the semi-colon and replaced it with a colon because otherwise, the slice variable would be declared at a global scope.